src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsProvider.cs

```diff
@@ -90,6 +97,11 @@ internal class LaunchSettingsProvider : OnceInitializedOnceDisposed, ILaunchSett
         [ImportMany]
         protected OrderPrecedenceImportCollection<ISourceCodeControlIntegration> SourceControlIntegrations { get; set; }
 
+        [ImportMany]
+        protected OrderPrecedenceImportCollection<IDefaultLaunchProfileProvider> DefaultLaunchProfileProviders { get; set; }
+
+        private readonly Lazy<ILaunchProfile?> _defaultLaunchProfile;
```

Comment:
This should go up with the other `private readonly` fields near the top.

Reply:
Fixed